### PR TITLE
Fix ScrollView inertia for the x axis bound region

### PR DESCRIPTION
**Pull request by jishnu7**
Fixes [#220](https://github.com/gameclosure/devkit/issues/220) in devkit.

### CODE REVIEW

**Comment by collingreen** [comment]
Thanks for this Jishnu -- I'll get to it as soon as I can.

**Comment by collingreen** [comment]
I haven't forgotten about this! Getting this tested and merged is still high on my todo list.

**Comment by jishnu7** [comment]
Easier way to reproduce the bug and test is, in adventuremap module, scroll to the left end or right end, and then do a up/down swipe.

File: src/ui/ScrollView.js [hunk 1]
```diff
@@ -453,39 +453,33 @@ exports = Class(View, function (supr) {
...
 				if (offset.x > bounds.maxX) {
-					distance = 0;
-				} else {
-					distance = delta.getMagnitude();
+					animateX = false;
 				}
 			}
 
-			if (distance) {
+			if (delta.getMagnitude() && (animateX || animateY)) {
```
Comment (rogueSkib) [inline src/ui/ScrollView.js]: @jishnu7 `distance` is already set above to `delta.getMagnitude()`, so we should use that instead here.

**Comment by rogueSkib** [comment]
I think with the minor change I mentioned above, and removing the additional file mode changes to the 3 files, this will be ready to merge.

**Comment by jishnu7** [comment]
Should I revert file mod changes? 755 looks wrong to me.

**Comment by rogueSkib** [comment]
Hmm, ya the change seems correct, just was odd to have it in this PR in particular. Going ahead with the merge, thanks!

**Pull request merged**